app/models/auth_sources/auth_source_internal.rb

```diff
@@ -1,4 +1,6 @@
 class AuthSourceInternal < AuthSource
+  scoped_search :on => :name, :complete_value => :true
```

Comment:
seems to be also defined in the AuthSource parent class, do we need to defined it here again?